Dockerfile

```diff
@@ -47,7 +47,10 @@ ENV ANDROID_HOME "/sdk"
 COPY tools/android-sdk-packages.txt tools/android-sdk-packages.txt
 COPY tools/install-android-sdk tools/install-android-sdk
 COPY tools/logging.sh tools/logging.sh
-RUN tools/install-android-sdk ${ANDROID_HOME}
+# Android sdkmanager does not have a quiet option and outputs highly
```

Comment:
Given that you are not redirecting stderr, we should ideally still see any failures. So, we should be ok on that front?

Reply:
Yes, that's my hope. I think I may have broken syntax, need to look a bit more.

Reply:
Looks ok! I'm leaving the comment in any case as my intent with it was to note that there could be stdout info level detail in the logging we omit that would be helpful. stderr will show something, hopefully, but it may be we need to re-run with stdout to really figure out details, who knows, so at least the comment aims to make that apparent.